src/nvim/window.c

```diff
@@ -3074,10 +3076,9 @@ int tabpage_index(tabpage_T *ftp)
  * FAIL.
  * Careful: When OK is returned need to get a new tab page very very soon!
  */
-static int 
-leave_tabpage (
-    buf_T *new_curbuf,        /* what is going to be the new curbuf,
-                                          NULL if unknown */
+static int leave_tabpage(
+    buf_T *new_curbuf,        // what is going to be the new curbuf,
+                              // NULL if unknown
     int trigger_leave_autocmds
 )
```

Comment:
Generally, stylistic changes make reviewing PRs like this harder, but if you want to make this look like neovim code...
```c
/// Prepare for leaving the current tab page.
/// 
/// When autocommands change "curtab" we don't leave the tab page and return false.
/// @warn When true is returned need to get a new tab page very very soon!
/// @param new_curbuf what is going to be the new curbuf or NULL if unknown
/// @oaram trigger_leave_autocmds If true, trigger leave autocomds.
static int leave_tabpage(buf_T *new_curbuf, int trigger_leave_autocmds) {
  ...
}
```
Been a while since I double checked doxygen syntax, and I didn't check the lines are under 80 chars, but this is basically how it should look.